pypose/module/dynamics.py

```diff
@@ -626,3 +626,42 @@ def c2(self):
         '''
         # Potential performance loss here - self.C and self.D involves jacobian eval
         return self._ref_g - bmv(self.C, self._ref_state) - bmv(self.D, self._ref_input)
+
+
+def toBTN(vec, T):
+    r'''
+    A helper function that reshape the input tensor of shape ``[..., n_dim]``
+    to ``[n_batch, n_timestep, n_dim]``.
+
+    Returns:
+        The reshaped tensor in shape of ``[B, T, N]``.
+    '''
+    if vec.ndim == 1:
+        vec = vec.unsqueeze(0)
+
+    if vec.ndim == 2:
+        vec = vec.unsqueeze(0)
+
+    if vec.shape[1] == 1:
+        vec = vec.repeat(1, T, 1)
+
+    return vec
+
+
+def runsys(system: System, T, x_traj, u_traj):
+    r'''
+    A helper function that runs the system for T steps given x and u trajectories or intial states.
```

Comment:
Run the system for T steps, given state trajectories (or initial state) and system input trajectories.